node_name: "rob498_drone_07"
offset: 0.05
land_offset: -0.3
launch_height: 1.5
vicon_enabled: false
rate: 20
vis: true
error_tol: 0.075

# PID: Linear
linear_x_Kp: 1.0
linear_x_Kd: 0.0
linear_x_Ki: 0.0

linear_y_Kp: 1.0
linear_y_Kd: 0.0
linear_y_Ki: 0.0

linear_z_Kp: 1.0
linear_z_Kd: 0.0
linear_z_Ki: 0.0

# PID: Angular
#angular_x_Kp: 1.0
#angular_x_Kd: 0.0
#angular_x_Ki: 0.0
#
#angular_y_Kp: 1.0
#angular_y_Kd: 0.0
#angular_y_Ki: 0.0

angular_z_Kp: 2.0
angular_z_Kd: 0.0
angular_z_Ki: 0.0
